app/src/androidTest/java/com/example/jasmiensofiecels/wordly/view/dailyWord/InvalidDailyWordActivityTest.java

```diff
@@ -0,0 +1,154 @@
+import android.util.Log;
+
+import com.example.jasmiensofiecels.wordly.R;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.RootMatchers.withDecorView;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+
+/**
+ * The purpose of this test class is to test the UI of the DailyWordActivity.
+ * These tests focus on invalid search words.
+ *
+ * Created by Jasmien Cels on 20/04/2018.
+ */
+
+@RunWith(JUnit4.class)
+public class InvalidDailyWordActivityTest {
+
+    private String invalidInput;
+    private String nullInput;
+    private Activity activity;
+
+    @Rule
+    public ActivityTestRule<DailyWordActivity> rule = new ActivityTestRule<>(DailyWordActivity.class);
+
+    @Before
+    public void initInvalidSearch() {
+        invalidInput = "invalidSpelling";
+        nullInput = "";
+        activity = rule.getActivity();
+    }
+
+    @Test
+    //The purpose of this test is to ensure that the search EditText text is set to the invalid word
+    public void searchWordIsInvalidWord() {
+
+        //When
+        onView(withId(R.id.search_word_et))
+                .perform(typeText(invalidInput), closeSoftKeyboard());
+        onView(withId(R.id.search_fab))
+                .perform(click());
+
+        try {
+            Thread.sleep(500);
+        } catch (Exception e) {
+            Log.d("Thread exception: ", e.getMessage());
+        }
+
+        //Then
+        onView(withId(R.id.search_word_et))
+                .check(matches(withText(invalidInput)));
+    }
+
+    @Test
+    //The purpose of this test is to ensure that the Toast appeared
+    public void toastAppearedOnInvalidInput() {
+
+        //When
+        onView(withId(R.id.search_word_et))
+                .perform(typeText(invalidInput), closeSoftKeyboard());
+        onView(withId(R.id.search_fab))
+                .perform(click());
+
+        try {
+            Thread.sleep(500);
+        } catch (Exception e) {
+            Log.d("Thread exception: ", e.getMessage());
+        }
+        //Then
+
+        onView(withText("Word not found")).
+                inRoot(withDecorView(not(is(activity.getWindow().getDecorView())))).
+                check(matches(isDisplayed()));
+    }
```

Comment:
Here's an interesting one. Try using an invalid input of `asd` instead of `invalidSpelling` and see what happens :)